pyrightconfig.json

```diff
@@ -7,7 +7,11 @@
     ],
     "exclude": [
         // test cases use a custom config file
-        "stubs/**/@tests/test_cases"
+        "stubs/**/@tests/test_cases",
+        // Bug in pyright that can't be ignored as it passes this one,
+        // but fails pyrightconfig.stricter.json. Can be removed after
+        // upgrading to 1.1.304 and removing ignores.
+        "stubs/tensorflow/tensorflow/io/__init__.pyi",
```

Comment:
Can this be removed now?

Reply:
Yes it can be removed. I'll cleanup pyright ignores/this later today.